src/constants.js

```diff
@@ -6,5 +6,8 @@ export default {
     copy: {
```

Comment:
@katjam The message "Not enough signal, voucher queued." can't come from the API right? Because it's shown when the user is offline and unable to reach the API.